### PR TITLE
Add fade in for new notifications

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: packrat/styles/metro/notices.css [hunk 1]
```diff
@@ -10,6 +10,9 @@
   padding: 10px 12px 10px 58px;
   min-height: 10px;
   background: #e5ebf1;
+  transition-property: background;
+  transition-duration: 3s;
```
Comment (andrewconner) [inline packrat/styles/metro/notices.css:14]: I haven't seen it visually, but do you think 3 seconds may be a long time?

Comment (gmethvin) [inline packrat/styles/metro/notices.css:14]: Danny said "a few seconds" and this looks okay so I made it 3s. Do you want to see?

Comment (2is10) [inline packrat/styles/metro/notices.css:14]: Personally, I'd go for 1.5s or so. Maybe we can talk Danny down over time (after he's seen it 100 times).

**Pull request merged**